tests/Feature/Store/SearchPageTest.php

```diff
@@ -363,6 +363,30 @@ public function itCanPaginateWithNumberedPages()
         ;
     }
 
+    /** @test */
+    public function itShowsCentreLabelsForUsersByDefault()
+    {
+        $this->markTestIncomplete(
```

Comment:
Sad but unavoidable until i get the selective mysql testing sorted.

Reply:
This is just one I haven't filled in yet.